Makefile

```diff
@@ -8,39 +8,39 @@ build:
 	mv main bench-routes
 
 view-v1.0:
-	cd dashboard/v1.0/ && sudo npm start
+	cd dashboard/v1.0/ && sudo yarn start
 
 view-v1.1:
-	cd dashboard/v1.1/ && sudo npm start
+	cd dashboard/v1.1/ && sudo yarn start
 
 test-views-v1.0:
-	cd dashboard/v1.0/ && npm install
-	cd dashboard/v1.0/ && npm run lint
-	cd dashboard/v1.0/ && npm run tlint
+	cd dashboard/v1.0/ && yarn add
+	cd dashboard/v1.0/ && yarn run lint
+	cd dashboard/v1.0/ && yarn run tlint
 	cd dashboard/v1.0/ && prettier '**/*.tsx' --list-different
-	cd dashboard/v1.0/ && npm run react-test
-	cd dashboard/v1.0/ && npm run react-build
-	cd dashboard/v1.0/ && npm run build
-	cd dashboard/v1.0/ && npm start &
+	cd dashboard/v1.0/ && yarn run react-test
+	cd dashboard/v1.0/ && yarn run react-build
+	cd dashboard/v1.0/ && yarn run build
+	cd dashboard/v1.0/ && yarn start &
 
 test-views-v1.1:
-	cd dashboard/v1.1/ && npm install
-	cd dashboard/v1.1/ && npm run lint
-	cd dashboard/v1.1/ && npm run tlint
+	cd dashboard/v1.1/ && yarn add
+	cd dashboard/v1.1/ && yarn run lint
+	cd dashboard/v1.1/ && yarn run tlint
 	cd dashboard/v1.1/ && prettier '**/*.tsx' --list-different
-	cd dashboard/v1.1/ && npm run test
-	cd dashboard/v1.1/ && npm run build
-	cd dashboard/v1.1/ && npm start &
+	cd dashboard/v1.1/ && yarn run test
+	cd dashboard/v1.1/ && yarn run build
+	cd dashboard/v1.1/ && yarn start &
 
 
 test-views-only:
```

Comment:
You can make different commands for `test-views-only` for v1.0 and v1.1, as there is a clash between yarn and npm

Like `test-views-only-v1.0` and `test-views-only-v1.1`

Reply:
Sorry. I didn't see it. I just replaced every word with yarn. Thanks for reviewing @ganeshpatro321 .